onnx/checker.cc

```diff
@@ -120,6 +120,7 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& /*ctx*/) {
     }
     return;
   } else {
+    bool isChecked = false;
```

Comment:
this is not needed, right?

Reply:
This is required to remove the default case